lib/Parser/Parse.cpp

```diff
@@ -5525,7 +5533,13 @@ void Parser::ParseFncDeclHelper(ParseNodeFnc * pnodeFnc, LPCOLESTR pNameHint, us
             }
         }
 
```

Comment:
Does the fact that we have a stub for a function necessarily mean that we want to defer it? I think we need to leave the deferral decision alone here and instead change the way parsing makes use of the stubs *if* we choose to defer.

Reply:
Agreed. I've pushed an update which leaves this decision in place.